pymongo/encryption.py

```diff
@@ -552,6 +554,128 @@ def __init__(
+            for the collection
+          - ``validationLevel`` (str): how strictly to apply the
+            validation rules to existing documents during an update.  The default level
+            is "strict"
+          - ``validationAction`` (str): whether to "error" on invalid documents
+            (the default) or just "warn" about the violations but allow invalid
+            documents to be inserted
+          - ``indexOptionDefaults`` (dict): a document specifying a default configuration
+            for indexes when creating a collection
+          - ``viewOn`` (str): the name of the source collection or view from which
+            to create the view
+          - ``pipeline`` (list): a list of aggregation pipeline stages
+          - ``comment`` (str): a user-provided comment to attach to this command.
+            This option is only supported on MongoDB >= 4.4.
+          - ``clusteredIndex`` (dict): Document that specifies the clustered index
+            configuration. It must have the following form::
+
+                {
+                    // key pattern must be {_id: 1}
+                    key: <key pattern>, // required
+                    unique: <bool>, // required, must be ‘true’
+                    name: <string>, // optional, otherwise automatically generated
+                    v: <int>, // optional, must be ‘2’ if provided
+                }
+          - ``changeStreamPreAndPostImages`` (dict): a document with a boolean field ``enabled`` for
+            enabling pre- and post-images.
+
+        .. versionadded:: 4.4
+           Added the codec_options, read_preference, and write_concern options.
+
+        .. _create collection command:
+            https://mongodb.com/docs/manual/reference/command/create
+        """
+        encrypted_fields = database._get_encrypted_fields(kwargs, name, False)
+        if encrypted_fields:
+            for field in encrypted_fields["fields"]:
+                if isinstance(field, dict) and field.get("keyId") is None:
+                    try:
+                        if kms_provider:
+                            field["keyId"] = self.create_data_key(
+                                kms_provider=kms_provider, **{data_key_opts or {}}
+                            )
+                    except WriteError as exc:
+                        raise EncryptionError(
+                            f"Error occured while creating data key with encryptedFields={str(encrypted_fields)}"
+                        ) from exc
+            kwargs["encryptedFields"] = encrypted_fields
+        else:
+            raise EncryptionError(
+                "'encryptedFields' must be provided as a keyword argument to create_encrypted_collection"
+                "or initialized with AutoEncryptionOpts."
+            )
+
+        return (
+            database.create_collection(
+                name=name,
+                **kwargs,
+            ),
+            encrypted_fields,
+        )
```

Comment:
I think this entire return could fit on one line.

Reply:
Fixed.